src/Build/BackEnd/BuildManager/BuildRequestData.cs

```diff
@@ -6,89 +6,16 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Linq;
+using System.Runtime.CompilerServices;
 using Microsoft.Build.Collections;
 using Microsoft.Build.Evaluation;
 using Microsoft.Build.Experimental.BuildCheck;
+using Microsoft.Build.Framework;
 using Microsoft.Build.Shared;
 
+[assembly: TypeForwardedTo(typeof(Microsoft.Build.Execution.BuildRequestDataFlags))]
```

Comment:
Let's move this to AssemblyInfo.cs - https://github.com/dotnet/msbuild/blob/main/src/Build/AssemblyInfo.cs